### ISSUE TITLE
testing: simplify i18n mock creation

### ISSUE THREAD

**Issue by jpommerening** [LaxarJS/laxar#447]
Currently, creating an i18n mock requires an actual i18n instance:

```js
const axI18nMock = createAxI18nMock( axI18n ); // :(
```

This is ok inside LaxarJS and bearable in widgets, but i18n instances are hard to come by when testing controls, leading to [reimplementation](https://github.com/LaxarJS/ax-input-control/blob/master/spec/helpers.js#L28) of the i18n functionality, making tests prone to mask problems when the i18n API or behaviour changes.

It would be really nice if one could instead create an i18n mock with:

```js
const axI18nMock = createAxI18nMock( axContextMock, axConfigurationMock ); // :)
```

(The need for this was identified in LaxarJS/laxar-mocks#50)

**Comment by x1B** [comment]
@jpommerening sure that we need a context mock? I would think an ideal axI18n mock would not even talk to the event bus. Instead, the test could call some API like `.mockUpdateLocale( name, tag )`?

**Comment by jpommerening** [comment]
Good point, agree 💯

**Comment by x1B** [comment]
Implemented on `master` (`v2.0.0`).

### BREAKING CHANGE

The API to `createAxI18nMock` of the `laxar/laxar-widget-service-mocks` library has changed. Previously, an axI18n-compatible object had to be passed in, effectively restricting use of the mock to the [LaxarJS Mocks](https://github.com/LaxarJS/laxar-mocks) library (which will now need to be adjusted).

Now, an axI18n mock can be created…

  - without arguments
  - passing a mapping of locale topic to language tag as the first argument.
     Use when `{ default: 'en' }` is not enough for testing
  - passing an AxContext-like object as the second argument (first argument may still be used, or set to an empty object). This should now be used by LaxarJS Mocks to share the axI18n eventBus with the remaining test
